cmd/upgrade/executor/options.go

```diff
@@ -77,41 +75,3 @@ func (u *UpgradeOptions) InitializeDefaults(cmd *cobra.Command) error {
 
 	return nil
 }
-
-// getUpgradePath gives the path for the upgrade
-func (u *UpgradeOptions) getUpgradePath() (string, error) {
-	podClient := pod.NewKubeClient()
-	from := strings.Split(u.fromVersion, "-")[0]
-	to := strings.Split(u.toVersion, "-")[0]
-	mayaLabels := "name=maya-apiserver"
-	mayaPods, err := podClient.WithNamespace(u.openebsNamespace).
-		List(
-			metav1.ListOptions{
-				LabelSelector: mayaLabels,
-			},
-		)
-	if err != nil {
-		return "", err
-	}
-	if len(mayaPods.Items) != 1 {
```

Comment:
TODO: For me need to check whether this check exists in different  PR #1510 or not